servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -335,6 +348,11 @@ def format_scattered_task(task_name, task_metadata):
                 failure_messages = [
                     f.get('message') for f in shard.get('failures')
                 ]
+            operation_details = None
+            if _should_get_operation_details(shard):
```

Comment:
If I'm understanding this right, you _always_ look up the operations details from Google (via rawls, via orchestration), even if the user never opens that modal?

Given that we're already battling to minimize (a) response times and (b) response sizes, would it be possible to make this operations fetching another endpoint in the shim, which is called if - but only if - the user tries to open the operations details panel?

Reply:
Does https://github.com/DataBiosphere/job-manager/pull/659#discussion_r299234778 address this?

Reply:
This one is more a general question - do we really want to make the call to rawls every time, even if the user never opens the modal? It seems likely to slow down the responsiveness of JM's workflow details page and increase the size of the response (and eg use more memory in the browser)

Reply:
I was reluctant to add the additional infrastructure to separate out this call without any evidence that it would have an effect on performance.  The only slowness I observed was after the job details page had been loaded and the JSON-formatting library was setting up particularly large contents of the operation details dialog (which would happen no matter what).

Reply:
How many tasks did you test with? Those requests look like they'll happen in series, and once per task.

eg this is the set of network requests that will now be needed to load a workflow:
```
[Once per workflow]  shim -> CromIAM -> Cromwell -> CromIAM -> shim
# then...
[Once per task]      shim -> Rawls -> PAPIv2 -> Rawls -> shim
```

And those "once per task" requests will happen in series, so every task in a scatter is presumably adding another set of network requests and response wait times?

Reply:
eg if that `[Once per task]` chain takes 100ms to complete, a workflow with a 100-wide scatter will have to make 100 of those request chains in series, and the time will presumably add up between them?